AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Device.java

```diff
@@ -159,29 +147,24 @@ public int getPhoneID() {
         return mPhoneID;
     }
 
+    private String getSimInformation(String simInfo, String errorMsg) {
+        try {
+            if(simInfo != null && !simInfo.isEmpty())
+                return simInfo;
+        } catch (Exception e) {
+            // SIM methods can cause Exceptions on some devices
+            log.error(String.format("%s %s", errorMsg, e.toString()));
+        }
+        return "N/A";
```

Comment:
I think we should make this string translateable.

In english it's `N/A` (Not available) but in german for example it should be 'N/V' (Nicht verfügbar).

If no context is available in order to get the translated String, you can change the return type from `String` to `Optional<String>`, so the caller knows that the requested information might not be present